### PR TITLE
Update to public versions

### PR DESCRIPTION
**Pull request by RussKie**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/extensions/pull/5749)

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -32,34 +22,14 @@
...
-      <package pattern="*" />
-    </packageSource>
-    <packageSource key="dotnet10-transport">
-      <package pattern="*" />
-    </packageSource>
     <packageSource key="richnav">
       <package pattern="*" />
     </packageSource>
   </packageSourceMapping>
-  <disabledPackageSources>
```
Comment (joperezr) [inline NuGet.config:57]: I think it should be better to keep the section, and just leave it empty as it is. That way, when we flow internal dependencies in preparation for our next release, maestro can still add sources bellow in the section that they should go.

File: eng/Version.Details.xml [hunk 2]
```diff
@@ -148,39 +148,39 @@
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
       <Sha>9d5a6a9aa463d6d10b0b0ba6d5982cc82f363dc3</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="9.0.1">
+    <Dependency Name="Microsoft.AspNetCore.App.Ref" Version="9.0.0">
```
Comment (joperezr) [inline eng/Version.Details.xml:151]: Out of curiosity, how did you change these versions? If you did it through darc, I would have expected the Sha's to change.

Comment (RussKie) [reply eng/Version.Details.xml:151]: I change those manually. Darc pulls the non-public versions (i.e., 9.0.1)

File: NuGet.config (hunk 1 above)
Comment (RussKie) [reply NuGet.config:57]: Maestro can add this section automatically, I believe

File: NuGet.config [hunk 3]
```diff
@@ -32,34 +22,14 @@
...
-    <packageSource key="dotnet10">
-      <package pattern="*" />
-    </packageSource>
-    <packageSource key="dotnet10-transport">
-      <package pattern="*" />
-    </packageSource>
     <packageSource key="richnav">
       <package pattern="*" />
     </packageSource>
   </packageSourceMapping>
```
Comment (RussKie) [inline NuGet.config:34]:
```suggestion
  </packageSourceMapping>
  <disabledPackageSources />
```

**Comment by RussKie** [comment]
Tested in https://github.com/dotnet/extensions-samples/pull/617

**Pull request merged**